### PR TITLE
Read from new output folder + fix outline layer

### PR DESCRIPTION
**Pull request by paulineribeyre**
Jira Ticket: COV-1078

PRC homepage maps:
- Load the chart images from the "generative_bayes_model" folder instead of the "bayes-by-county" folder
- The new "cases" and "rt" charts are now `svg` instead of `png`
- Remove the "deaths" charts which are not generated anymore
- Only display the "county-highlight" map layer when the "Case Data" layer is selected. Display a "county-outline" layer for all Illinois counties with a thinner outline, for all layers.

### CODE REVIEW

**Review by ocshawn — APPROVED** [review]
looks good

**Comment by paulineribeyre** [comment]
waiting for sponsor/PM approval

**Review by ocshawn — APPROVED** [review]

**Pull request merged**
